internal/pkg/cli/task_run.go

```diff
@@ -370,6 +367,11 @@ func (o *runTaskOpts) shouldPromptForAppEnv() bool {
 
 // Execute deploys and runs the task.
 func (o *runTaskOpts) Execute() error {
+	if o.groupName == "" {
+		dir, _ := os.Getwd()
```

Comment:
Should we catch this err and just error out? I think it's fine to error here since they can always specify the a value with the flag in case they have some weird issue
```suggestion
		dir, err := os.Getwd()
		if err != nil {
			log.Errorf("Could not retrieve working directory, please use --%s to specify a task group name.\n", taskGroupNameFlag)
			return fmt.Errorf("get working directory: %v", err) 
		}
```